crane/views/v2.py

```diff
@@ -63,6 +65,12 @@ def name_redirect(relative_path):
     if not base_url.endswith('/'):
         base_url += '/'
 
+    # perform base_url rewrite if relevant CDN config present
```

Comment:
i suggest to move this into a separate method since name_redirect() is already a huge monster